scripts/demos/common/telemetry.sh

```
@@ -1,45 +1,28 @@
-    fi
-    if [[ -f "$TELEMETRY_DIR/profiles/$profile/otlp-collector.yml" ]]; then
-      collector_configs+=("--config=/etc/otlp-collector-$profile.yml")
-    fi
-  done
-  # Exported for override compose files (e.g. a demo layer) whose command must restate the
-  # collector configs contributed by the active profiles, since Docker Compose replaces the
-  # command list rather than appending to it.
   export TELEMETRY_COLLECTOR_CONFIGS="${collector_configs[*]}"
   if [[ -n "${TELEMETRY_COMPOSE_OVERRIDE_FILE:-}" ]]; then
     [[ -f "$TELEMETRY_COMPOSE_OVERRIDE_FILE" ]] || die "telemetry compose override not found: $TELEMETRY_COMPOSE_OVERRIDE_FILE"
     export TELEMETRY_COMPOSE_OVERRIDE_DIR="$(cd "$(dirname "$TELEMETRY_COMPOSE_OVERRIDE_FILE")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)